otx/algorithms/classification/configs/efficientnet_v2_s_cls_incr/template_supcon.yaml

```diff
@@ -0,0 +1,64 @@
+# Description.
```

Comment:
As I mentioned, you should remove template_*.yaml

Reply:
Ah, that parameter can be set via the command line?